test/broadcast.jl

```diff
@@ -223,12 +223,11 @@ let x = sin.(1:10)
     @test sin.(atan2.(x, 3.7)) == broadcast(x -> sin(atan2(x,3.7)), x)
     @test atan2.(x, 3.7) == broadcast(x -> atan2(x,3.7), x) == broadcast(atan2, x, 3.7)
 end
-# Use side effects to check for loop fusion.  Note that, due to #17314,
-# a broadcasted function is currently called an extra time with an argument 1.
+# Use side effects to check for loop fusion.
 let g = Int[]
     f17300(x) = begin; push!(g, x); x+1; end
```

Comment:
Probably this should be changed to e.g. `x+2`, since with `x+1` it will now give the same results whether the loops are fused or not.